client/packages/admin/src/index.ts

```diff
@@ -389,7 +391,7 @@ class InstantAdmin<Schema = {}> {
    *   )
    */
```

Comment:
nit: would we want to add `deprecated` + a note about these functions?

Reply:
Note: will deprecate global `tx` down the road